ChangeLog

```diff
@@ -1,6 +1,7 @@
+	* fix invalid torrent state finish_duration calculation
```

Comment:
@arvidn here it is
https://github.com/arvidn/libtorrent/pull/1492/commits/c71611785d90ba301746b26725e40f93d12d7dfe#diff-19929347e18a6ff1fc37f09ec720306bL8259

there was a failure at timebase shift
You may want to backport this fix to RC_1_1